src/Modal.js

```diff
@@ -156,6 +160,19 @@ const Modal = React.createClass({
     if (this.props.keyboard && e.keyCode === 27) {
       this.props.onRequestHide();
     }
+  },
+
+  focusModalContent () {
+    this.lastFocus = document.activeElement;
```

Comment:
iframes aren't that high on my list of concerns, I was just following the other code.